### PR TITLE
Remove minimum interval check and let the API reject too-frequent requests.

### PR DESCRIPTION
**Pull request by igorpeshansky**
Also restore `collectd`'s default interval of 10s (overridden it in our packaged configuration anyway).

This reverts commit 734db20639b4f0a4bd68ae39bbdb9b4f2c70e4d8 ("Disallow intervals that are less than sixty seconds").

### CODE REVIEW

File: src/daemon/collectd.h [hunk 1]
```diff
@@ -227,7 +227,7 @@
 #endif
 
 #ifndef COLLECTD_DEFAULT_INTERVAL
-#define COLLECTD_DEFAULT_INTERVAL 60.0
+#define COLLECTD_DEFAULT_INTERVAL 10.0
```
Comment (bmoyles0117) [inline src/daemon/collectd.h:230]: Please update the description to include that the default interval is being updated to 10 seconds.

**Review by bmoyles0117 — CHANGES_REQUESTED** [review]

File: src/daemon/collectd.h (hunk 1 above)
Comment (igorpeshansky) [reply src/daemon/collectd.h:230]: That's just the default collectd interval. Our packaged config still uses 60s. I figured putting all that in the PR description would have been somewhat misleading... WDYT?

Comment (bmoyles0117) [reply src/daemon/collectd.h:230]: I would suggest removing this change, if you don't intend to communicate it in the PR description.

Comment (igorpeshansky) [reply src/daemon/collectd.h:230]: Removing it would not make this a clean revert. Added to the PR description instead. PTAL.

**Review by bmoyles0117 — APPROVED** [review]
LGTM

**Pull request merged**